frontend/express/public/core/app-resolution/templates/app-resolution.html

```diff
@@ -18,15 +18,15 @@ <h2> {{i18n('resolutions.title')}} </h2>
 		<cly-section>
 			<div class="bu-columns bu-is-gapless bu-mt-2 technology-pie-graphs">
 				<div class="bu-column bu-is-6">
-					<cly-chart-pie :option="pieOptionsNew" :showToggle="false"></cly-chart-pie>
+					<cly-chart-pie :option="pieOptionsNew" :showToggle="false" v-loading="isLoading" :force-loading="isLoading"></cly-chart-pie>
```

Comment:
I can't understand why we use `v-loading="isLoading"` and `:force-loading="isLoading"` together :slightly_smiling_face:

Reply:
Because there is no "isLoading" prop similar to the datatable in the charts, we send the loading state to the component with the :force-loading prop. Thus, the empty state doesn't show in the cases for the api is running.
And for compatibility, we decided the prop name with Prikshit 🙂